dm/worker/subtask.go

```diff
@@ -435,6 +435,22 @@ func (st *SubTask) setResult(result *pb.ProcessResult) {
 	st.result = result
 }
 
+// markResultCanceled mark result as canceled if stage is Paused.
+// This func is used to pause a task which has been paused by error,
```

Comment:
we should also support "pause a task which has no error, just the user want to pause it" 🤔

Reply:
I don't understand what you mean. We currently support pause-task manually for a normal task.